### PR TITLE
[10.0][MIG] crm_helpdesk

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Migration of crm_helpdesk.

Please, review.

I didn't remove the `.idea` folder because it was impossible. Everytime I delete it, it gets automatically created again with the `workspace.xml` 😓

Finally, I proceeded to push the `.idea` changes in separated commits, in case someone wants to cherrypick my migration.

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
.idea issue is fixed, I'll review ASAP

File: crm_helpdesk/models/crm_helpdesk.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+              \nIf the case is in progress the status is set to \'Open\'.\
+              \nWhen the case is over, the status is set to \'Done\'.\
+              \nIf the case needs to be reviewed then the status is set to'
+             ' \'Pending\'.', default='draft')
+
+    @api.multi
+    def on_change_partner_id(self, partner_id):
+        values = {}
+        if partner_id:
+            partner = self.env['res.partner'].browse(partner_id)
```
Comment (AaronHForgeFlow) [inline crm_helpdesk/models/crm_helpdesk.py]: I'd do `self.email_from = partner.email`

File: crm_helpdesk/README.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+and categorize your interventions with a channel and a priority level.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Miquel Raïch <miquel.raich@eficent.com>
```
Comment (AaronHForgeFlow) [inline crm_helpdesk/README.rst]: Odoo should be here, right?

**Review by AaronHForgeFlow — COMMENTED** [review]
Few remarks

File: crm_helpdesk/README.rst (hunk 2 above)
Comment (MiquelRForgeFlow) [reply crm_helpdesk/README.rst]: Added.

**Comment by LoisRForgeFlow** [comment]
Proposed to OCA/crm

**Pull request closed without merging**